TheWidlarzGroup/react-native-video
# Support preventsDisplaySleepDuringVideoPlayback

- [x] Test description
- [x] General description

## Description

This flag allows the device display to sleep while playing a video. Useful if video is not primary content in the app.

```jsx
<Video
  ...
  preventsDisplaySleepDuringVideoPlayback={ false }
  ...
/>
```

## Test description
To test this
1. instantiate with `preventsDisplaySleepDuringVideoPlayback={false}`
2. Run on device (on iOS it should not be attached to Xcode while running)
3. Wait and let the display sleep.

Closes #1869
